diff --git a/modules/canopy/lang/lambda/companion/pkg.generated.mbti b/modules/canopy/lang/lambda/companion/pkg.generated.mbti
index c7570986..06ea457b 100644
--- a/modules/canopy/lang/lambda/companion/pkg.generated.mbti
+++ b/modules/canopy/lang/lambda/companion/pkg.generated.mbti
@@ -32,6 +32,8 @@ pub fn get_lambda_dot_resolved(@editor.SyncEditor[@ast.Term]) -> String raise Fa
 
 pub fn get_lambda_resolution(@editor.SyncEditor[@ast.Term]) -> @lambda.Resolution raise Failure
 
+pub fn minimal_no_shadow_p1_annotations(Array[@eval.EvalResult], @core.ProjNode[@ast.Term], @core.SourceMap) -> Map[@core.NodeId, Array[@protocol.ViewAnnotation]]
+
 pub fn new_lambda_editor(String, capture_timeout_ms? : Int, parent_runtime? : @cells.Runtime) -> (@editor.SyncEditor[@ast.Term], LambdaCompanion) raise
 
 pub fn parse_tree_edit_op(Json) -> @edits.TreeEditOp raise @editor.TreeEditError
